src/utils/diff/index.ts

```diff
@@ -0,0 +1,66 @@
+import * as jsdiff from 'diff';
+import { colorLines } from './colors';
+import { type, emptyRepresentation, cleanUp } from './util';
+import { INDENT } from './const';
+
+
+function unifiedDiff (actual: string, expected: string): string {
+    const msg = jsdiff.createPatch('string', actual, expected);
+    const lines = msg.split('\n').splice(5);
+
+    return (
+        colorLines('diff added', '+ expected') +
```

Comment:
Ideally, all UI strings should be stored in a single location. Let's refactor this to a template stored in the `src/notifications`.

Reply:
Addressed it by moving UI strings to the error template